docs/reference/getting-started.asciidoc

```diff
@@ -147,6 +147,19 @@ And now we are ready to start our node and single cluster:
 ./elasticsearch
 --------------------------------------------------
 
+=== Installation example with Homebrew
+
+--------------------------------------------------
+brew install elasticsearch
```

Comment:
I don’t think we need full details on how to use Homebrew here, a comment saying that Elasticsearch can be installed via Honebrew is sufficient.